content/2017/12/12/introducing-herman.rst

```diff
@@ -0,0 +1,389 @@
+public: yes
+tags: [Herman, CSS, Sass, 'Design Systems', Code, 'Style Guides']
+author: miriam
+image:
+  - src: 'blog/2017/herman-intro/header.jpg'
+headline:
+  - tagline: 'An agile approach to design systems'
+summary: |
+  **Design systems streamline development, communication, and consistency** --
+  but often rely on dedicated teams and extended budgets.
+  We wanted a tool to help us create and maintain
+  living style guides & pattern libraries
+  in an agile process, and on a budget.
+  `Herman`_ keeps our development flow simple,
+  and our UX consistent,
+  as we iterate on patterns
+  and scale over time.
```

Comment:
Mention that Herman is a free, open-source tool.
Perhaps change "our" and "we" to "you" and "your" so people realize this is tool for them to use, not just an internal tool OddBird made for ourselves.

Reply:
To me, using "we" makes it more of a testimonial: this thing has *worked* for us. I think there are better ways to clarify that this tool is available, if that's really not clear yet.